src/main/java/com/klarna/hiverunner/HiveRunnerRule.java

```diff
@@ -62,16 +61,16 @@ public Statement apply(Statement base, Description description) {
 
     class HiveRunnerRuleStatement extends Statement {
 
-        private Object target;
-        private Statement base;
-        private Path testBaseDir;
-        private StandaloneHiveRunner runner;
+        private final Object target;
```

Comment:
Let's just do a reformat for now, not introduce all kinds of other IntelliJ suggested improvements, otherwise this MR becomes too big with too many changes.